client/api/src/client.rs

```diff
@@ -87,7 +87,55 @@ pub struct ClientInfo<Block: BlockT> {
 	/// Best block hash.
 	pub chain: Info<Block>,
 	/// State Cache Size currently used by the backend
-	pub used_state_cache_size: Option<usize>,
+	pub usage: UsageInfo,
+}
+
+/// Memory statistics for client instance.
+#[derive(Default, Clone, Debug)]
+pub struct MemoryInfo {
+	/// Size of state cache.
+	pub state_cache: usize,
+	/// Size of backend database cache.
+	pub database_cache: usize,
+}
+
+/// I/O statistics for client instance.
+#[derive(Default, Clone, Debug)]
+pub struct IoInfo {
```

Comment:
Are these stats persisted between restarts of the client? If not, we might want the docs to say that this only for the duration since last start to avoid confusion down the line?

Reply:
Not persisted

Sure, I'll add a comment

Reply:
Added comment to `UsageInfo`